subprojects/core/src/integTest/groovy/org/gradle/testfixtures/ProjectBuilderEndUserIntegrationTest.groovy

```diff
@@ -29,18 +29,23 @@ class ProjectBuilderEndUserIntegrationTest extends AbstractIntegrationSpec {
     def setup() {
         buildFile << """
         apply plugin: 'groovy'
+        apply plugin: 'jvm-test-suite'
```

Comment:
Groovy Plugin applies Java Plugin applies JVM Test Suite plugin implicitly.  But I guess no harm being explicit here.

Reply:
I was not aware of this. It might be worth removing the [example](https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html#sec:jvm_test_suite_usage) of that from the docs as it is really not necessary at all, and I think having a big code block advertising it can be misleading (as it was for me here).